server/src/test/java/org/cloudfoundry/identity/uaa/security/web/UaaRequestMatcherTests.java

```diff
@@ -67,6 +69,26 @@ void pathMatcherMatchesExpectedPathsAndAcceptHeaderNull() {
         assertThat(matcher.matches(request("/somePath", null))).isTrue();
     }
 
+    @ParameterizedTest
```

Comment:
can you check sonar
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=3276&id=cloudfoundry-identity-parent&open=AZTWuNP2sTkMRfYdBil0

and maybe merge it into one test